huaweicloud/services/acceptance/iam/resource_huaweicloud_identity_access_key_test.go

```diff
@@ -26,7 +25,7 @@ func getIdentityAccessKeyResourceFunc(c *config.Config, state *terraform.Resourc
 	}
 
 	if found.AccessKey != state.Primary.ID {
-		return nil, fmtp.Errorf("Access Key not found")
+		return nil, fmt.Errorf("Access Key not found")
```

Comment:
```suggestion
		return nil, fmt.Errorf("access Key not found")
```

Reply:
the **Access Key** maybe more suitable as it is a technical terms.